StartBonusActivation/orchestrators.ts

```diff
@@ -0,0 +1,135 @@
+import { DurableOrchestrationClient } from "durable-functions/lib/src/durableorchestrationclient";
+import { left, right } from "fp-ts/lib/Either";
+import {
+  fromEither,
+  fromLeft,
+  TaskEither,
+  taskEither,
+  tryCatch
+} from "fp-ts/lib/TaskEither";
+import {
+  IResponseErrorForbiddenNotAuthorized,
+  IResponseErrorInternal,
+  IResponseSuccessAccepted,
+  ResponseErrorForbiddenNotAuthorized,
+  ResponseErrorInternal,
+  ResponseSuccessAccepted
+} from "italia-ts-commons/lib/responses";
+import { FiscalCode, NonEmptyString } from "italia-ts-commons/lib/strings";
+import {
+  isOrchestratorRunning,
+  makeStartBonusActivationOrchestratorId,
+  makeStartEligibilityCheckOrchestratorId
+} from "../utils/orchestrators";
+
+import { toString } from "fp-ts/lib/function";
+import { readableReport } from "italia-ts-commons/lib/reporters";
+import { InstanceId } from "../generated/definitions/InstanceId";
+
+/**
+ * Check if the provided user has a pending activation request.
+ * If there's no pending requests right(false) is returned
+ * @param client
+ * @param fiscalCode
+ *
+ * @returns either false or a custom response indicating whether there's a process running or there has been an internal error during the check
+ */
+export const checkBonusActivationIsRunning = (
```

Comment:
this code will be obsoleted by the new way we're gonna do this check

Reply:
cc @francescopersico

Reply:
if it's not a big issue, I'd like to merge this change asap to avoid conflicts with @francescopersico - then I'll go ahead with the tests